assets/js/modules/analytics/setup.js

```diff
@@ -818,6 +841,25 @@ class AnalyticsSetup extends Component {
 							</h5>
 						</div>
 					</div>
+					{ onSettingsPage && useSnippet && ! ampClientIDOptIn && ( ! ampEnabled || ( ampEnabled && ampMode !== 'primary' ) ) && (
```

Comment:
This can simply be `onSettingsPage && useSnippet && ampMode !== 'primary'` - not sure how the AMP client ID got in here, but that shouldn't matter at all :)

Reply:
Ah, ten-four! Thanks 😄